fastlane/lib/fastlane/actions/docs/capture_android_screenshots.md

```diff
@@ -52,7 +52,25 @@ Ensure that the following permissions exist in your **src/debug/AndroidManifest.
 
 ##### Configuring your <a href="#ui-tests">UI Tests</a> for Screenshots
 
-1. Add `@ClassRule public static final LocaleTestRule localeTestRule = new LocaleTestRule();` to your tests class to handle automatic switching of locales
+1. Add `LocaleTestRule` to your tests class to handle automatic switching of locales. If you're using Java use:
+
+```java
+@ClassRule
+public static final LocaleTestRule localeTestRule = new LocaleTestRule();
+```
+If you're using Kotlin use:
+```kotlin
+@Rule @JvmField
+val localeTestRule = LocaleTestRule()
+```
+Important is the `@JvmField` annotation. It won't work like that:
+```kotlin
+companion object {
+    @get:ClassRule
+    val localeTestRule = LocaleTestRule()
+}
+```
```

Comment:
nit: if you indent these by 3 spaces, they will be put inside the `1.` listing in the preview. Not super sure if this also works in our rendered version, but might.